app/src/app/list/polls-table/polls-table.component.sass

```diff
@@ -3,28 +3,45 @@
 @import '../../../sizes'
 
 table
+  display: grid
+  grid-template-columns: 1fr min-content min-content min-content min-content
   border: solid 1px $maincolor
   border-collapse: collapse
+
+
+
```

Comment:
Three empty lines is too much. If you want to keep one empty line, then please unindent one level on what follow.